.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Forem DEVcontainer",
+  "dockerComposeFile": "../docker-compose.yml",
+  "service": "devcontainer",
+  "runServices": ["postgres", "redis"],
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+  "remoteEnv": {
+    "LOCAL_WORKSPACE_FOLDER": "${localWorkspaceFolder}",
+    "GIT_EDITOR": "code --wait"
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {
+      "version": "latest",
+      "enableNonRootDocker": "false",
+      "moby": "true"
```

Comment:
This is here so that the workspace will be provided with docker, which then gives the VS Code docker extension proper functionality